### DISCUSSION TITLE
Java -> expected Error Fixing

### DISCUSSION THREAD

**Discussion by NaveedAzmi** [community/community#39109] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hi Respected Member's 

I'm Naveed From Pakistan, and I'm self learner Java Programming. I'm Beginner Stage of Learning, I face some Issue my Practices Code Kindly anyone can Help me to solve my Issue
 I shall be Thankful to you

**Comment by david-engelmann** [comment, marked as answer]
[The Java section of Stackoverflow is probably a great place to ask a specific question on the error!](https://stackoverflow.com/tags/java/info)

**Reply by NaveedAzmi** [reply]
Thank You So Much, Dear David, but my problem still not fix can you help me More on this issue????

**Reply by david-engelmann** [reply]
Absolutely!

**Reply by NaveedAzmi** [reply]
[AddressBook.zip](https://github.com/community/community/files/10014717/AddressBook.zip)

This is My Practices Code

**Comment by NaveedAzmi** [comment]
Thank You So Much, Dear David, but my problem still not fix
